doc/ale-supported-languages-and-tools.txt

```diff
@@ -120,6 +120,7 @@ Notes:
 * CSS
   * `VSCode CSS language server`
   * `cspell`
+  * css-beautify
```

Comment:
Please add the backticks "`" around as the other entries.

Reply:
Thanks for the hint. It is done.